manifests/plugin/tasks.pp

```diff
@@ -7,19 +7,17 @@
 # $package:: Package name to install, use ruby193-rubygem-foreman-tasks on Foreman 1.8/1.9 on EL
 #            type:String
 #
-# $service:: Service name
-#            type:String
+# $dynflow_in_core:: Whether Foreman ships the Dynflow executor service (true on 1.15+)
+#                    type:Boolean
 #
 class foreman::plugin::tasks(
   $package = $foreman::plugin::tasks::params::package,
-  $service = $foreman::plugin::tasks::params::service,
+  $dynflow_in_core = $::foreman::dynflow_in_core,
 ) inherits foreman::plugin::tasks::params {
   foreman::plugin { 'tasks':
     package => $package,
-  } ~>
-  service { 'foreman-tasks':
-    ensure => running,
-    enable => true,
-    name   => $service,
+  }
+  unless $dynflow_in_core {
```

Comment:
My point is: if it's true, it's included anyway (through ```foreman::service```). Here you also ensure it's included if it's false. Doesn't that mean you can unconditionally include it?